### PR TITLE
Remove max_axial_mesh_size from ABTR workflow example

### PR DESCRIPTION
**Pull request by shikhar413**
Refs #435

### CODE REVIEW

**Comment by shikhar413** [comment]
@GiudGiud doing some more cleanup of Griffin, where I am removing a parameter that is no longer needed (see https://github.inl.gov/ncrc/griffin/pull/2118). This causes a VTB test failure, and I'm not sure whether this PR should be merged first or the one in Griffin

**Comment by GiudGiud** [comment]
First merge in griffin, then do a Griffin submodule update in this PR

File: doc/content/sfr/abtr_xsgen_workflow/abtr_xsgen_workflow.md [hunk 1]
```diff
@@ -17,6 +17,7 @@
                        computing_needs:HPC
                        cross_sections:MCC3
                        fiscal_year:2024
+                       sponsor:NEAMS
```
Comment (GiudGiud) [inline doc/content/sfr/abtr_xsgen_workflow/abtr_xsgen_workflow.md:20]: adding this one already in my PR

I think you may want to add "cross section generation" to the input_features?

Comment (shikhar413) [reply doc/content/sfr/abtr_xsgen_workflow/abtr_xsgen_workflow.md]: Reverted last commit and added cross_section_generation to input_features

**Comment by shikhar413** [comment]
@GiudGiud tests are now passing with griffin sub module update and using !include syntax

**Review by GiudGiud — APPROVED** [review]

**Comment by GiudGiud** [comment]
looks like Josh merged a breaking change into Griffin first. I might have to process his PR first

**Comment by shikhar413** [comment]
No worries, this can be merged after his

**Comment by GiudGiud** [comment]
either that or pick a submodule before his PR on adding a terminator for negative power in simulations

**Comment by shikhar413** [comment]
Rebasing and re-pushing in case it's necessary after Josh's PR

**Comment by GiudGiud** [comment]
Tests passed on previous run

**Pull request merged**